### DISCUSSION TITLE
1.8 startup mode saved?

### DISCUSSION THREAD

**Discussion by snydoodlsh** [djphazer/O_C-Phazerville#98] [Q&A] [answered]
Is there way for my OC to startup in the same mode each time it powers up? Right now, I have to launch hemispheres each time it powers up (Plum Audio VOR version). I thought saving the state in hemispheres would do that, but it doesn't.

**Comment by djphazer** [comment, marked as answer]
You must do a Long-press save on the main menu to set the current App as the default on startup!

**Comment by PramudithaN** [comment]
@snydoodlsh Yes, you can definitely set it to boot straight into Hemispheres!

Saving the state inside Hemispheres (like saving a preset) doesn't always tell the global O_C firmware to boot into that app. To save your startup app and global settings to the EEPROM, you need to do a global save from the main menu.

Here is how to do it:

From inside Hemispheres, long-press the Right Encoder (for about 2 seconds) to exit back to the main O_C App Selection menu.
While in the main App Selection menu (with Hemispheres highlighted), long-press the Right Encoder again.
You should see a little confirmation graphic (like an expanding square/quadrilateral) pop up on the screen.
This confirms that the module has saved its global state to the EEPROM. The next time you power down and power back up, it should boot directly into Hemispheres!
